.tests/integration/config/resources.yaml

```
@@ -3,7 +3,158 @@
+  mem_mb: 4096
+  mem_per_cpu: 4096
+
+gatk_sam_to_fastq:
+  mem_mb: 5120
+  mem_per_cpu: 5120
+
+gatk_sort_sam:
+  mem_mb: 4096
+  mem_per_cpu: 4096
+
+gatk_mark_duplicates:
```

YAMLlint (1.38.0)

[error] duplication of key "gatk_mark_duplicates" in mapping

(key-duplicates)